docs/how-to-guides/setup-thirdparty-auth-for-grafana.md

```diff
@@ -0,0 +1,101 @@
+# How to setup third party OAuth for Grafana?
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Steps](#steps)
+* [What's next?](#whats-next)
+
+## Introduction
+
+This document explains how to enable any supported auth provider on Grafana deployed as a part of Prometheus Operator.
+
+## Prerequisites
+
+- On Packet: You have a DNS entry in any DNS provider for `grafana.mydomain.net` against the Packet EIP.
```

Comment:
I think it would be good to clarify, if user should have prometheus operator already installed, or we **install it with  third party auth support**.

Reply:
What do you mean? Can you elaborate on this? The doc is such that it does not matter if user has prometheus-operator installed or not they will eventually install it in this doc.

Reply:
If we instruct a user to create a configuration file, then we assume it's a fresh installation IMO.

Reply:
added a OR to the create file line.

Reply:
Thanks, looks good!